package.json

```diff
@@ -1,32 +1,65 @@
 {
-  "name": "mento-sdk",
-  "version": "1.0.0",
-  "description": "The SDK for Mento Protocol",
-  "main": "index.js",
+  "name": "@mentoprotocol/sdk",
+  "version": "0.1.0",
+  "license": "MIT",
+  "author": "Mento Labs",
+  "main": "dist/index.js",
+  "module": "dist/sdk.esm.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
   "scripts": {
-    "lint": "yarn prettier",
-    "lint:check": "yarn prettier:check && yarn lint:check",
-    "postinstall": "husky install",
-    "prettier": "prettier --config \"./.prettierrc.yml\" --write \"**/*.{json,md,ts,js,yml}\"",
-    "prettier:check": "prettier --config \"./.prettierrc.yml\" --check \"**/*.{json,md,ts,js,yml}\""
+    "analyze": "size-limit --why",
+    "build": "dts build",
+    "lint": "dts lint",
+    "prepare": "dts build",
+    "size": "size-limit",
+    "start": "dts watch",
+    "test": "dts test",
+    "coverage": "dts test --coverage"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/mento-protocol/mento-sdk.git"
+  "husky": {
+    "hooks": {
+      "pre-commit": "dts lint"
+    }
   },
-  "author": "Mento Labs",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/mento-protocol/mento-sdk/issues"
+  "prettier": {
+    "printWidth": 80,
+    "semi": false,
+    "singleQuote": true,
+    "trailingComma": "es5"
+  },
+  "jest": {
+    "testEnvironment": "node"
+  },
+  "peerDependencies": {},
+  "engines": {
+    "node": ">=12"
+  },
+  "size-limit": [
+    {
+      "path": "dist/sdk.cjs.production.min.js",
+      "limit": "10 KB"
+    },
+    {
+      "path": "dist/sdk.esm.js",
+      "limit": "10 KB"
+    }
+  ],
+  "devDependencies": {
+    "@size-limit/preset-small-lib": "^8.1.0",
+    "@tsconfig/recommended": "^1.0.1",
+    "dts-cli": "^1.6.0",
```

Comment:
Haven't heard of dts before. It seems nice but not very popular. Consider that the building/bundler will have full control of the outputted code and it's nearly impossible to detect malicious payloads a rogue tool like this can sneak in. For that reason, I'd recommend avoiding this and configuring TS manually. Depending on who the consumers of this are, you probably don't even need a bundler, just the TS output is probably good enough! I can help with that if you need

Reply:
Agreed. Let's keep it lean. We can merge this PR like this and revert the change in another ticket.

Reply:
Great point on malicious payloads, will keep it like this for now as Bogdan already created a separate ticket and do it there